.gitignore

```diff
@@ -5,4 +5,5 @@
 /Manifest.toml
 /dev/
 /docs/build/
```

Comment:
need to ignore the generated css file too

Reply:
I didn't find the css file. Where is it?